_includes/integrations/angular-tech-ref.md

```diff
@@ -30,6 +31,16 @@
   - [`allowedEvents`](#allowedevents)
   - [`ignoreEvents`](#ignoreevents)
 
+### Supported Angular versions
+
+The following table shows the supported versions of Angular and the required version of the `tinymce-angular` integration package.
+
+| Angular Version | `tinymce-angular` version |
+|:---:|:---:|
+| 9+ | 4.x |
+| 6 to 8 | 3.x |
```

Comment:
This seems incorrect... I'm 99% sure this should be `5 to 8` as 3.x always supported that range previously and then we have the `@tinymce/tinymce-angular-legacy` package for 4.x.

As a side note, should the `@tinymce/tinymce-angular-legacy` package also be mentioned below? as I find it rather confusing to say not supported and then have the page this be used in say `Angular 4+ integration`

Reply:
Hmm... I missed you changed the title to 6+. However there are still other places that refer to using the legacy package for Angular 4.

Reply:
Just checked the source, we only mentioned the legacy package once (the line I removed)

Reply:
That still doesn't address the incorrect versions listed there... also more needs to happen if we're going to drop the "legacy" package (not from the docs side, but for example the package needs to be marked deprecated, etc...).

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1820/commits/2097dc66603756fc01c1945969537ffaa7233aef